doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2069,7 +2070,7 @@ in :ref:`canonicalstructures`; here only a simple example is given.
 
       See :ref:`canonicalstructures` for a more realistic example.
 
-   .. cmdv:: Canonical {? Structure } @ident {? : @type } := @term
+   .. cmdv:: {? Local | #[local] } Canonical {? Structure } @ident {? : @type } := @term
```

Comment:
You show the 2 forms of the attribute here but not in the `Canonical @qualid` command above, is that intentional?

Reply:
Yes, because the other command only supports one version (the attribute)

Reply:
We actually parse Local as the attribute.

Reply:
Oh, I see. Fixed